packages/fiori-generator-shared/src/npm-package-scripts/types.ts

```diff
@@ -38,4 +38,6 @@ export interface PackageScriptsOptions {
     localStartFile?: string;
     /** If true, a script for starting the app without flp will be generated. Defaults to true. */
     generateIndex?: boolean;
+    /** If true, the generated scripts will include search parameters. Defaults to true. */
```

Comment:
Same comment, not clear what gets added or why.

Reply:
Updated description